ssh/ssh.go

```diff
@@ -90,9 +92,24 @@ func RemoteSSHCommand(user string, privateKey string, host string, port string,
 		}
 	}
 
-	log.Info("Running command %s", cmd)
-	err = session.Run(cmd)
-	if err != nil {
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
```

Comment:
SIGHUP signifies the controlling terminal is closed & is normally used by daemons to signify reloading config, not stopping.

Happy to handle syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT though. Note that [as documented](https://golang.org/pkg/os/#Signal) only SIGINT & SIGKILL are guaranteed to exist on all platforms.